docs/dev-guide/src/getting-started/building.md

```diff
@@ -13,11 +13,14 @@ not by installing your distro's `rust` package which will most likely be too old
 
 For the installation you can refer to [these instructions](https://www.rust-lang.org/tools/install).
 
-Once you have a working rust installation, you'll need to add the `wasm32-unknown-unknown` target and install the nightly toolchain
+Once you have rustup, you'll need to do the following.
+1. Install rust version 1.76.0. Creditcoin uses the feature `stdsimd` which was removed in later Rust versions.
+2. Default to using the nightly toolchain corresponding to Rust version 1.76.0 as shown below.
+3. Add the `wasm32-unknown-unknown` target
 
 ```bash
-rustup update
-rustup toolchain install nightly
+rustup install 1.76.0
```

Comment:
Why do we need to install a stable version and then install the nightly version, I don't understand it?

Reply:
You're right. We don't need to. 👍  That was an artifact of me trying to figure out why builds were failing.